Dockerfile

```diff
@@ -14,19 +14,8 @@
 #   --tmpfs /tmp --tmpfs /run --tmpfs /run/lock \
 #   -v /sys/fs/cgroup:/sys/fs/cgroup:ro \
 #
-# We also pre-install our packages, so that tests run faster.
-FROM ubuntu:20.04
+ARG BASE=jrei/systemd-ubuntu:20.04
```

Comment:
This could get out of date. With the nice Make targets, you could probably get away with not having a sensible default. I have had good experience with providing a self-documenting, error-inducing default like `ARG BASE=you-must-set-the-base-image`.

Reply:
Great idea, will add.